reslib/pom.xml

```diff
@@ -47,10 +47,10 @@
 									<goal>wget</goal>
 								</goals>
 								<configuration>
-									<url>https://github.com/liferay/liferay-portal/raw/0f75db5b5c6a8a7c6b07295cc12acdf362bfb1b4/portal-web/third-party/alloy-${alloyui.zip.version}.zip</url>
+									<url>https://github.com/liferay/liferay-portal/raw/6.2.x/portal-web/third-party/alloy-${alloyui.zip.version}.zip</url>
```

Comment:
Well the problem isn't the name of the file, it's that the md5 is static and is referenced below. Since the md5 is static, if the file is changed/upgraded, the build will break.